samcli/lib/build/app_builder.py

```diff
@@ -611,3 +629,73 @@ def _parse_builder_response(stdout_data: str, image_name: str) -> Dict:
             raise ValueError(msg)
 
         return cast(Dict, response)
+
+    @staticmethod
+    def _get_env_vars_value(filename: Optional[str]) -> Any:
```

Comment:
This does the same thing:

https://github.com/aws/aws-sam-cli/blob/b5cfc46aa9726b5cd006df8ecc08d1b4eedeb9ea/samcli/commands/local/cli_common/invoke_context.py#L400

I'm not convinced the duplication or special method is necessary, instead of a generic `json.load` wrapper.

Reply:
sounds good, removed this method for simplicity